screens/Request/ReceiveVcScreen.strings.json

```diff
@@ -2,5 +2,12 @@
   "header": "{{vcLabel}} details",
   "save": "Save {{vcLabel}}",
   "verifyAndSave": "Verify and save",
-  "discard": "Discard"
```

Comment:
reject label default value? Is it being used?

Reply:
I am not sure why it wasn't there. Probably Discard is used instead of Reject. Anyhow to keep the consistency adding the reject translation.